csi/identity.go

```diff
@@ -24,8 +24,8 @@ import (
 )
 
 const (
-	// CSI 1.4 compatible
-	csiDriverVersion = "1.4.0"
```

Comment:
Nomad shows this version in the CLI output, so we need to have the correct version exposed here via GetPluginInfo